### ISSUE TITLE
get_the_terms, get_the_term_list, wp_get_post_terms broken

### ISSUE THREAD

**Issue by pabloselin** [not-only-code/qtranslate-slug#22]
It seems that when you call a term via the above function I'm only getting the first term link correctly and the other ones return the correct taxonomy but the post permalink instead of the term slug. 

Sometimes I'm not getting any of the taxonomy links correct.

Deactivating the plugin the problem disappears...

**Comment by not-only-code** [comment]
What version of wordpress you have?

**Comment by pabloselin** [comment]
I'm using version 3.4.1

**Comment by not-only-code** [comment]
Two questions more, is your installation multisite? And, witch plugins are you using too?

**Comment by pabloselin** [comment]
I'm not using multisite, in the plugins department i'm using:

AutoChimp
Google XML Sitemaps for Qtranslate
Qtranslate (2.5.31)
Qtranslate extension (another extension that adds multilanguage input via javascript to some fields.)
Qtranslate slug (your plugin)
Redirection
Regenerate Thumbnails

You can watch the problem first hand in this site: 

http://www.ong-ngo.org/fr/opinion-3/2013-tunis-to-become-the-capital-of-solidarity/

When you check the tags aside in the right you will notice that the tag links are wrong. 

I will try to do more research about the problem later so if I find anything strange or if I can fix the problem I'll tell you.

Thanks!

**Comment by pabloselin** [comment]
Hi, I reproduced the issue in a blank wordpress website and I think it's a bug.
Here are the steps I followed in a blank wordpress with default theme and only qtranslate and qtranslate_slug activated.
1. Create a new post and put some custom slugs in the available languages.
2. Create a new tag
3. Save the post.
4. Go to the tags administration page and look for your recently created tag.
5. You will see that the tag slug is the same as the slug of the post you just edited.

So the problem is not that the functions are broken with your plugin but that it's assigning incorrect values to the tag slug.
